docs/getting-started-scalardb.md

```diff
@@ -0,0 +1,344 @@
+# Getting Started with Helm Charts (Scalar DB Server)
+
+This document explains how to get started with Scalar DB Server using Helm Chart in your test environment. Here, we assume that you already have a Mac or Linux environment for testing.  
+
+## Tools
+
+In this guide, we will use the following tools for testing.  
+
+1. Docker
+1. minikube
+1. kubectl
+1. Helm
+
+## Environment
+
+In this guide, we will create the following environment in your local by using Docker and minikube.  
+
+```
+On the Docker Network (named minikube)
+
+                   +-----------------------+
+  +-----------+    | +------------------+  |    +-----------+
+  |           |    | | Scalar DB Server |  |    |           |
+  |  Client   |    | +------------------+  |    | Cassandra |
+  | Container |    |                       |    | Container |
+  |           |    | Kubernetes (minikube) |    |           |
+  +-----+-----+    +-----------+-----------+    +-----+-----+
+        |                      |                      |
+*-------+----------------------+----------------------+-------*
+             ------->                     ------->
+               gRPC                    Cassandra Driver
+```
+
+## Step 1. Install tools
+
+First, you need to install the following tools that will be used in this guide.  
+
+1. Install the Docker according to the [Docker document](https://docs.docker.com/engine/install/)  
+
+1. Install the minikube according to the [minikube document](https://minikube.sigs.k8s.io/docs/start/)  
+
+1. Install the kubectl according to the [Kubernetes document](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)  
+
+1. Install the helm command according to the [Helm document](https://helm.sh/docs/intro/install/)  
+
+## Step 2. Start minikube with docker driver
+
+We need to start the Cassandra container as backend storage of the Scalar DB Server on the same network of the minikube, so you need to be started the minikube with docker driver.
```

Comment:
```suggestion
```
I don't fully get the sentence, but do we need it? 
It seems confusing to talk about C* when setting up minikube and docker driver.

Reply:
This sentence explains about `docker network` to make Pods (on minikube) and Cassandra communicate properly.
However, it seems that it leads some confusing...
Also, since it will work properly by copy and paste, I think that users don't need to understand it (about docker network) to use this guide.

So, I apply this suggestion (delete this sentence).